### PR TITLE
[#751] UI Data Binding

### PR DESCRIPTION
**Pull request by michal-szostak**
## What's in this PR?

This PR contains

* Toggling visibility of the sidebar, categories
* Bind product description to the UI component
* Bind sentinel scene name to search results
* Remove size label from sentinel search list

Closes: #751

### CODE REVIEW

File: s4e-web/src/app/views/map-view/map-view.component.html [hunk 1]
```diff
@@ -165,13 +161,10 @@
...
   <div class="dropdown__content dropdown__content--product">
-    <span class="dropdown__header">Opis produktu</span>
+    <span class="dropdown__header" i18n>Opis produktu</span>
     <div class="dropdown__close dropdown" (click)="toggleProductDescription(false)"></div>
-    <h2>Setvak i chmurki</h2>
-    <div class="dropdown__description">
-      <p>
-        Miejsce na opis prodktu. W tym momencie jest statyczne.
-      </p>
+    <h2>{{(activeProducts$ | async).displayName}}</h2>
```
Comment (danielkryska) [inline s4e-web/src/app/views/map-view/map-view.component.html:166]: missing i18n

File: s4e-web/src/app/views/map-view/map-view.component.html [hunk 2]
```diff
@@ -165,13 +161,10 @@
...
+    <div class="dropdown__description" [innerHTML]="(activeProducts$ | async).description">
```
Comment (danielkryska) [inline s4e-web/src/app/views/map-view/map-view.component.html:167]: missing i18n

**Review by danielkryska — APPROVED** [review]

File: s4e-web/src/app/views/map-view/sentinel-search/search-results/search-results.component.html [hunk 3]
```diff
@@ -40,10 +40,6 @@
                 <span i18n>Instrument:</span>
                 <strong>{{result.instrument}}</strong>
               </li>
-              <li>
-                <span i18n>Size:</span>
-                <strong>{{result.size}}</strong>
-              </li>
```
Comment (jswk) [inline s4e-web/src/app/views/map-view/sentinel-search/search-results/search-results.component.html:46]: Not only should size be removed, but `spacecraft` and `processing_level` metadata should be shown dynamically here

File: s4e-web/src/app/views/map-view/sentinel-search/search-result-modal/search-result-modal.component.html [hunk 4]
```diff
@@ -2,7 +2,7 @@
 <button [disabled]="isLast$ | async" (click)="nextResult()" class="modal__next"><i class="fa fa-chevron-right"></i></button>
 <s4e-generic-modal [buttonX]="true" [modalId]="registeredId">
   <div class="s4e-modal-header">
-    {{(searchResult$ | async)?.id}}
+    {{(searchResult$ | async)?.sceneKey}}
```
Comment (jswk) [inline s4e-web/src/app/views/map-view/sentinel-search/search-result-modal/search-result-modal.component.html:5]: I think the value displayed here should be only the last part of the path, excluding `.scene`, as discussed.
For example: /MSG_Products_WM/OST/20200201/**202002010000_WV-IR**.scene (the bold part should be retained)

**Review by abacz — APPROVED** [review]

**Review by jswk — DISMISSED** [review]
When the sidebar is toggled, then the timeline and the info button don't resize correctly, see:
image
Moreover, between viewport widths of 701px and 1050px the user button is over the zoom buttons.
image
Below width 1050px also the toggle sidebar button doesn't work. It's impossible to see the available layers.

File: s4e-web/src/app/views/map-view/sentinel-search/search-results/search-results.component.html [hunk 5]
```diff
@@ -22,7 +22,7 @@
       <li *ngFor="let result of searchResults" data-test="search-result-entry">
         <div class="search-result__package">
           <span class="sentinel-type">{{result.instrument}}</span>
-          <strong class="package-name">S1A_IW_GRDH_1SDV_20200518T052552_20200518T052617_032615_03C710_C6F8</strong>
+          <strong class="package-name">{{result.sceneKey}}</strong>
```
[the remark this answers is not in the document]
Comment (michal-szostak) [reply s4e-web/src/app/views/map-view/sentinel-search/search-results/search-results.component.html:25]: sounds good, are those files always ending with `.scene`?

[the remark this answers is not in the document]
Comment (jswk) [reply s4e-web/src/app/views/map-view/sentinel-search/search-results/search-results.component.html:25]: Yes, always ending with `.scene`

**Pull request merged**